### DISCUSSION TITLE
Support for Google Cloud Structured Logging

### DISCUSSION THREAD

**Discussion by simon-graham** [meilisearch/product#743] [Feedback & Feature Proposal]
The ```experimental_logs_mode``` JSON log format is very useful. However, for running on cloud platforms like GCP it would be very helpful if Melisearch could output JSON in GCP's specific required format. E.g. it uses ```severity: "ERROR"``` rather than ```level: "ERROR"```. Meilisearch logs would then be much easier to query, view, create alerts for, etc, in GCP.

There's an example of how to do this in Rust here: https://github.com/ralpha/google_cloud_logging/

The log format is described here: https://cloud.google.com/logging/docs/structured-logging

**Comment by macraig** [comment]
Hi @simon-graham , apologies for the late reply. Although we currently don't have the bandwidth to work on this, we do see the value it would bring to users. We're open to external contributors adding it to the [experimental logs feature](https://github.com/orgs/meilisearch/discussions/723), if that's something you'd be interested in.
